sunpy/net/jsoc/tests/test_jsoc.py

```diff
@@ -53,6 +53,8 @@ def test_return_query_args(client):
                         a.jsoc.Segment('Bp') & a.jsoc.Segment('magnetogram'))
     # Because res.query_args is list that contains dict
     assert 'primekey' in res.query_args[0]
+    assert "magnetogram" in res.keys()
+    assert res["magnetogram"][0].startswith("/SUM")
```

Comment:
How has this changed the results table a user sees?

Reply:
It adds a new column with the name of the segment.